platformio_tasmota32.ini

```diff
@@ -52,6 +52,9 @@ build_flags                 = ${esp_defaults.build_flags}
                               -Dmemcmp_P=memcmp
                               ;for TLS we can afford compiling for 4K RSA keys
                               -DUSE_4K_RSA
+                              -I$PROJECT_DIR/include
+                              -include "fix_esp32c3.h"

```

Comment:
Including "fix_esp32c3.h" is not needed for non-C3. But it does not harm either.

Reply:
I wrote the same. Placing here avoids accidently forgetting